Gems/Terrain/Code/Source/TerrainRenderer/TerrainFeatureProcessor.cpp

```diff
@@ -76,26 +77,24 @@ namespace Terrain
 
     void TerrainFeatureProcessor::Initialize()
     {
-        {
-            // Load the terrain material asynchronously
-            const AZStd::string materialFilePath = "Materials/Terrain/DefaultPbrTerrain.azmaterial";
-            m_materialAssetLoader = AZStd::make_unique<AZ::RPI::AssetUtils::AsyncAssetLoader>();
-            *m_materialAssetLoader = AZ::RPI::AssetUtils::AsyncAssetLoader::Create<AZ::RPI::MaterialAsset>(materialFilePath, 0u,
-                [&](AZ::Data::Asset<AZ::Data::AssetData> assetData, bool success) -> void
+        // Load the terrain material asynchronously
+        const AZStd::string materialFilePath = "Materials/Terrain/DefaultPbrTerrain.azmaterial";
```

Comment:
Should we expose this on the Terrain World Renderer component as a material picker to pick the terrain material?

Reply:
For the terrain as a whole, I don't think so, unless you mean as a very temporary measure. The properties of the material will be dictated by component properties and data in the terrain system. For instance for V0 terrain, the macro color and normal should be provided by a macro material component. The range that detail textures fade out should probably be render level component. Where detail textures render will be controlled by gradients. We'll probably swap out which materials different sectors render with under the hood for performance.

One exception (long term at least) is the detail textures should be real materials so they can built in the material editor with visual feedback. We won't use those materials directly to render the terrain, but can pull properties off of them to feed into different parts of the terrain rendering system.

Reply:
Ok, sounds good.  If the editable properties themselves end up exposed on the component, this makes sense to me.  If the properties were meant to be edited in the Material Editor, then I would've wanted to be able to select which material I'm using, so that I could have different terrain materials with different property configurations set up.

Reply:
yeah the whole thing with having the properties on the material is just a temporary unblock you thing 😄